openmdao/jacobians/assembled_jacobian.py

```diff
@@ -366,6 +366,9 @@ def _update(self, system):
         if ext_mtx is not None:
             ext_mtx._post_update()
 
+        if self._under_complex_step:
+            self._int_mtx._matrix = int_mtx._matrix.astype(np.complex)
```

Comment:
does this conversion to complex also need to be done for ext_mtx?

Reply:
Not that I know of. The Direct solver uses int_mtx directly, but not ext_mtx.

Reply:
but if you were using block-lin-GS it would need to be converted... right?